common-service-core/src/test/java/org/zowe/apiml/security/HttpsFactoryTest.java

```diff
@@ -44,11 +50,12 @@ public void shouldCreateSecureSslSocketFactory() {
     }
 
     @Test
-    public void shouldCreateIgnoringSslSocketFactory() {
+    public void shouldCreateIgnoringSslSocketFactory() throws KeyStoreException {
         HttpsConfig httpsConfig = httpsConfigBuilder.verifySslCertificatesOfServices(false).build();
         HttpsFactory httpsFactory = new HttpsFactory(httpsConfig);
         ConnectionSocketFactory socketFactory = httpsFactory.createSslSocketFactory();
         assertEquals(SSLConnectionSocketFactory.class, socketFactory.getClass());
+        assertFalse(httpsFactory.getUsedKeyStore().aliases().hasMoreElements());
```

Comment:
What about the second case with ignoring=false for purpose of restTemplateWithoutKeystore etc. I think we don't have any tests for that.

Reply:
It makes great sense. I have added the test case.